packages/frontend/src/hooks/useGetSetIntersection.ts

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+export const useGetSetIntersection = () => {
+  // NOTE: eventually this should use caching on the backend
```

Comment:
yeah the json files are large (the largest 15MB) so better to cache it in the future 👍

Reply:
yep yep